src/components/Menu/MenuItem.tsx

```diff
@@ -153,18 +225,69 @@ class MenuItem extends UIComponent<Extendable<MenuItemProps>, MenuItemState> {
           ...accessibility.keyHandlers.root,
         },
         overrideProps: () => ({
-          children: menuItemInner,
+          children: (
+            <>
+              {menuItemInner}
+              {maybeSubmenu}
+            </>
+          ),
+          onClick: this.handleClick,
+          onBlur: this.handleWrapperBlur,
         }),
       })
     }
     return menuItemInner
```

Comment:
Yes, because the submenu as an element is part of the wrapper (it cannot be part of the anchor tag, which is the element created if the wrapper is not provided). I agree that this is not obvious, maybe we should add that in the doc somewhere.